### PR TITLE
Fixed the missing <DependentUpon> tags in project files.

### PR DESCRIPTION
**Pull request by lextm**
This is to address #403. The code base seems to come from .NET Framework 1.1 where the necessary tags are missing in project files.

### CODE REVIEW

File: src/managed/OpenLiveWriter.ApplicationFramework/OpenLiveWriter.ApplicationFramework.csproj [hunk 1]
```diff
@@ -259,15 +259,19 @@
       <SubType>UserControl</SubType>
     </Compile>
     <EmbeddedResource Include="ApplicationForm.resx">
+      <DependentUpon>ApplicationForm.cs</DependentUpon>
       <SubType>Designer</SubType>
     </EmbeddedResource>
     <EmbeddedResource Include="ApplicationManager.resx">
+      <DependentUpon>ApplicationManager.resx</DependentUpon>
```
Comment (ScottIsAFool) [inline src/managed/OpenLiveWriter.ApplicationFramework/OpenLiveWriter.ApplicationFramework.csproj]: Is this correct? All the rest are dependent upon cs files, this one is a resx.

**Comment by lextm** [comment]
@ScottIsAFool fixed.

**Comment by kathweaver** [comment]
I've downloaded setup.exe and have published a blog post.  Nothing seems to have been broken.

**Comment by kathweaver** [comment]
Now I feel dumb, because I can open the satellite form just fine, but I get the same old error when I try to open PostEditorForm.  Do similar changes need to be made on that?

**Comment by lextm** [comment]
@kathweaver That's a separate issue I am going to raise.

**Pull request merged**